### ISSUE TITLE
Please update bower.json to not depend on a specific version of d3. 

### ISSUE THREAD

**Issue by pmcelhaney** [Caged/d3-tip#148]
```js
  "dependencies": {
    "d3": "3.5.5"
  }
```

should be:

```js
  "dependencies": {
    "d3": "^3.5.5"
  }
```

So that we can avoid this problem when running `bower install`:

```
Unable to find a suitable version for d3, please choose one:
    1) d3#3.5.5 which resolved to 3.5.5 and is required by d3-tip#0.6.7
    2) d3#^3.5.6 which resolved to 3.5.16 and is required by my-project
```

**Comment by pmcelhaney** [comment]
If you're busy, I can send a PR, and all you have to do is hit the merge button. I just don't want to add to the 21 open pull requests if there's no one around to merge them. :)

**Comment by vdh** [comment]
Fixed for d3 v4 with the tag [v0.7.0](https://github.com/Caged/d3-tip/tree/v0.7.0), and for d3 v3 with the tag [v0.6.8](https://github.com/Caged/d3-tip/tree/v0.6.8). Unfortunately I don't have bower/npm access so I can't make official releases.
